linera-service/src/linera.rs

```diff
@@ -762,6 +761,40 @@ enum ProjectCommand {
     Test { path: Option<PathBuf> },
 }
 
+#[derive(StructOpt)]
+struct JsonArg {
+    /// JSON file specifying initialization arguments, passed to the contract's `initialize` method on the chain
+    /// that creates the application. (But not on other chains, when it is registered there.)
+    ///
+    /// Cannot be used with `--json-args`.
+    #[structopt(long)]
+    file_args: Option<PathBuf>,
+    /// JSON file specifying initialization arguments, passed to the contract's `initialize` method on the chain
+    /// that creates the application. (But not on other chains, when it is registered there.)
+    ///
+    /// Cannot be used with `--file-args`.
+    #[structopt(long)]
+    json_args: Option<String>,
+}
+
+impl JsonArg {
+    fn into_bytes(self) -> Result<Vec<u8>, anyhow::Error> {
+        match (self.file_args, self.json_args) {
+            (Some(_), Some(_)) => bail!("cannot have both a json string and file"),
+            (Some(file), None) => {
+                let file_contents = fs::read_to_string(file)?;
+                let as_value: Value = serde_json::from_str(&file_contents)?;
+                Ok(serde_json::to_vec(&as_value)?)
+            }
+            (None, Some(json)) => {
+                let as_value: Value = serde_json::from_str(&json)?;
+                Ok(serde_json::to_vec(&as_value)?)
+            }
+            (None, None) => bail!("json arguments must be provided"),
```

Comment:
```suggestion
            (None, None) => bail!("JSON arguments must be provided"),
```

Alternatively, we _could_ make them optional?

Reply:
Indeed, I don't think we have consensus on this yet so I'm leaving it for a future discussion 😅